File: web-front/src/main/java/com/isystk/sample/web/front/controller/api/v1/entry/remind/EntryRemindRestForm.java
package com.isystk.sample.web.front.controller.api.v1.entry.remind;

import java.io.Serializable;
import javax.validation.constraints.Email;
import javax.validation.constraints.NotBlank;
import javax.validation.constraints.Size;

class EntryRemindRestForm implements Serializable {

  private static final long serialVersionUID = 7593564324192730932L;

  /**
   * メールアドレス
   */
  @NotBlank
  @Email
  @Size(max = 100)
  String email;

  public @NotBlank @Email @Size(max = 100) String getEmail() {
    return this.email;
  }

  public void setEmail(@NotBlank @Email @Size(max = 100) String email) {
    this.email = email;
  }
}

File: web-front/src/main/kotlin/com/isystk/sample/web/front/controller/api/v1/entry/remind/EntryRemindRestForm.kt
package com.isystk.sample.web.front.controller.api.v1.entry.remind

import lombok.Getter
import lombok.Setter
import java.io.Serializable
import javax.validation.constraints.Email
import javax.validation.constraints.NotBlank
import javax.validation.constraints.Size

@Setter
@Getter
class EntryRemindRestForm : Serializable {
    /**
     * メールアドレス
     */
    var email: @NotBlank @Email @Size(max = 100) String? = null

    companion object {
        private const val serialVersionUID = 7593564324192730932L
    }
}